brokerapi/brokers/cloudsql/broker.go

```diff
@@ -690,15 +687,15 @@ func updateOperationId(instance models.ServiceInstanceDetails, operationId strin
 	}
 	instance.OtherDetails = string(otherDetails)
 
-	if err = db_service.SaveServiceInstanceDetails(&instance); err != nil {
+	if err = db_service.SaveServiceInstanceDetails(ctx, &instance); err != nil {
 		return fmt.Errorf(`Error saving instance details to database: %s. WARNING: this instance cannot be deprovisioned through cf.
 		Please contact your operator for cleanup`, err)
 	}
 	return nil
 }
 
 // LastOperationWasDelete checks if the last async operation was a deletion (as opposed to a provision).
-func (b *CloudSQLBroker) LastOperationWasDelete(instanceId string) (bool, error) {
+func (b *CloudSQLBroker) LastOperationWasDelete(ctx context.Context, instanceId string) (bool, error) {
 	op, err := db_service.GetLastOperation(instanceId)
```

Comment:
Should probably pass it through to GetLastOperation as well.

Reply:
Thanks, I forgot we had one call that wasn't auto-generated :-)